examples/create-react-app/src/App.js

```diff
@@ -1,25 +1,56 @@
 import React from 'react';
-import logo from './logo.svg';
+import { useEffect, useState } from 'react';
 import './App.css';
 
 function App() {
+  const [date, setDate] = useState(null);
+  useEffect(() => {
+    async function getDate() {
+      const res = await fetch('/api/date');
+      const newDate = await res.text();
+      setDate(newDate);
+    }
+    getDate();
+  }, []);
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
+    <main>
+      <h1>Create React App + Go API</h1>
+      <h2>
+        Deployed with{' '}
         <a
-          className="App-link"
-          href="https://reactjs.org"
+          href="https://zeit.co/docs"
           target="_blank"
-          rel="noopener noreferrer"
+          rel="noreferrer noopener"
         >
-          Learn React
+          ZEIT Now
         </a>
-      </header>
-    </div>
+        !
+      </h2>
+      <p>
+        <a
+          href="https://github.com/zeit/now-examples/tree/master/create-react-app-functions"
```

Comment:
As @styfle mentioned:

```suggestion
          href="https://github.com/zeit/now-examples/tree/master/create-react-app"
```